accounts/views.py

```diff
@@ -89,3 +89,16 @@ class MakeDepositView(TemplateTitleMixin, TransactionView):
 
     def get_success_url(self) -> str:
         return reverse_lazy('accounts:detail', kwargs={"number": self.object.number})
+
+class MakeWithdrawView(TemplateTitleMixin, TransactionView):
```

Comment:
Could you please add `CurrentYearMixin` as an inherited class?

We are using this mixin to display the current year in the template footer.

You can see your usage at the `CreateAccountView()` class.

Reply:
Also, it would be good to be added into `MakeDepositView` and keep this logic in all future views using this same `accounts/base.html` template.

Reply:
Everything's fixed! Could you please review again?